src/styles.css

```
@@ -708,3 +708,73 @@ footer a:hover{text-decoration:underline}
+  #searchInput {
+    width: 100% !important;
+    min-width: 0 !important;
+  }
+  #surpriseBtn {
+    width: 100% !important;
+  }
+}
+
+/* C: Org card grid breakpoints */
+@media (max-width: 900px) {
+  #orgsGrid {
```

Stylelint (17.11.1)

[error] Expected id selector "`#orgsGrid`" to be kebab-case (selector-id-pattern)

(selector-id-pattern)